styles/config/vocabularies/Technical/accept.txt

```diff
@@ -224,6 +230,7 @@ Rollup
 [Rr]untime[s]?
 rsync
 Rsyslog
+[Ss]afetensors
```

Comment:
is "safetensors" both capital *and* lowercase? if the project itself is internally inconsistent, let's pick one version and stick to it, and enforce that version with vale.

Reply:
It's all over the place - safetensor, Safetensor, SafeTensor. Their [docs](https://huggingface.co/docs/safetensors/en/index) use "safetensor". We can pick that.